Allow user to set BLAS_LIBS for own blas libraries and mimic standard AX_BLAS
autoconf macro behaviour.

bicatali@gentoo.org

--- starpu-1.0.1/configure.ac.orig	2012-05-29 23:30:19.000000000 +0100
+++ starpu-1.0.1/configure.ac	2012-05-30 00:02:22.000000000 +0100
@@ -1434,6 +1434,11 @@
      if test x$use_system_blas = xyes; then
         AC_DEFINE(STARPU_SYSTEM_BLAS, [1], [use refblas library])
 	blas_lib=system
+     elif test x"$BLAS_LIBS" != x; then
+        AC_DEFINE(STARPU_SYSTEM_BLAS, [1], [use user defined library])
+	STARPU_BLAS_LDFLAGS="$BLAS_LIBS"
+	AC_SUBST(STARPU_BLAS_LDFLAGS)
+	blas_lib=system
      else
 	blas_lib=none
      fi
